--- a/prometheus/summary_test.go
+++ b/prometheus/summary_test.go
@@ -289,6 +289,7 @@
 }
 
 func TestSummaryDecay(t *testing.T) {
+	t.Skip("Skipping flaky test.")
 	if testing.Short() {
 		t.Skip("Skipping test in short mode.")
 		// More because it depends on timing than because it is particularly long...
--- a/model/signature_test.go
+++ b/model/signature_test.go
@@ -248,6 +248,7 @@
 }
 
 func TestEmptyLabelSignature(t *testing.T) {
+	t.Skip("Skipping flaky test.")
 	input := []map[string]string{nil, {}}
 
 	var ms runtime.MemStats
