--- a/prometheus/registry.go
+++ b/prometheus/registry.go
@@ -33,7 +33,7 @@
 	"strings"
 	"sync"
 
-	"bitbucket.org/ww/goautoneg"
+	"github.com/prometheus/client_golang/deps/bitbucket.org/ww/goautoneg"
 	"code.google.com/p/goprotobuf/proto"
 
 	dto "github.com/prometheus/client_model/go"
--- a/prometheus/summary.go
+++ b/prometheus/summary.go
@@ -21,7 +21,7 @@
 	"sync"
 	"time"
 
-	"github.com/beorn7/perks/quantile"
+	"github.com/prometheus/client_golang/deps/github.com/beorn7/perks/quantile"
 	"code.google.com/p/goprotobuf/proto"
 
 	dto "github.com/prometheus/client_model/go"
--- a/Godeps/_workspace/src/github.com/beorn7/perks/quantile/example_test.go
+++ b/Godeps/_workspace/src/github.com/beorn7/perks/quantile/example_test.go
@@ -10,7 +10,7 @@
 	"strconv"
 	"time"
 
-	"github.com/beorn7/perks/quantile"
+	"github.com/prometheus/client_golang/deps/github.com/beorn7/perks/quantile"
 )
 
 func Example_simple() {
