--- a/extraction/metricfamilyprocessor.go
+++ b/extraction/metricfamilyprocessor.go
@@ -18,7 +18,7 @@
 	"io"
 	"math"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 
--- a/prometheus/counter_test.go
+++ b/prometheus/counter_test.go
@@ -17,7 +17,7 @@
 	"math"
 	"testing"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func TestCounterAdd(t *testing.T) {
--- a/prometheus/desc.go
+++ b/prometheus/desc.go
@@ -11,7 +11,7 @@
 
 	"github.com/golang/protobuf/proto"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/prometheus/client_golang/model"
 )
--- a/prometheus/example_selfcollector_test.go
+++ b/prometheus/example_selfcollector_test.go
@@ -18,7 +18,7 @@
 
 	"github.com/golang/protobuf/proto"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
--- a/prometheus/examples_test.go
+++ b/prometheus/examples_test.go
@@ -23,7 +23,7 @@
 	"sort"
 	"time"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/golang/protobuf/proto"
 
--- a/prometheus/expvar_test.go
+++ b/prometheus/expvar_test.go
@@ -19,7 +19,7 @@
 	"sort"
 	"strings"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
--- a/prometheus/gauge_test.go
+++ b/prometheus/gauge_test.go
@@ -20,7 +20,7 @@
 	"testing"
 	"testing/quick"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func listenGaugeStream(vals, result chan float64, done chan struct{}) {
--- a/prometheus/go_collector_test.go
+++ b/prometheus/go_collector_test.go
@@ -5,7 +5,7 @@
 	"testing"
 	"time"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func TestGoCollector(t *testing.T) {
--- a/prometheus/histogram.go
+++ b/prometheus/histogram.go
@@ -23,7 +23,7 @@
 	"github.com/golang/protobuf/proto"
 
 	"github.com/prometheus/client_golang/model"
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 // A Histogram counts individual observations from an event or sample stream in
--- a/prometheus/histogram_test.go
+++ b/prometheus/histogram_test.go
@@ -22,7 +22,7 @@
 	"testing"
 	"testing/quick"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func benchmarkHistogramObserve(w int, b *testing.B) {
--- a/prometheus/http_test.go
+++ b/prometheus/http_test.go
@@ -19,7 +19,7 @@
 	"testing"
 	"time"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 type respBody string
--- a/prometheus/metric.go
+++ b/prometheus/metric.go
@@ -16,7 +16,7 @@
 import (
 	"strings"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 // A Metric models a single sample value with its meta data being exported to
--- a/prometheus/registry.go
+++ b/prometheus/registry.go
@@ -36,7 +36,7 @@
 	"github.com/prometheus/client_golang/deps/bitbucket.org/ww/goautoneg"
 	"github.com/golang/protobuf/proto"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/prometheus/client_golang/model"
 	"github.com/prometheus/client_golang/text"
--- a/prometheus/registry_test.go
+++ b/prometheus/registry_test.go
@@ -26,7 +26,7 @@
 	"testing"
 
 	"github.com/golang/protobuf/proto"
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 type fakeResponseWriter struct {
--- a/prometheus/summary.go
+++ b/prometheus/summary.go
@@ -24,7 +24,7 @@
 	"github.com/prometheus/client_golang/deps/github.com/beorn7/perks/quantile"
 	"github.com/golang/protobuf/proto"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/prometheus/client_golang/model"
 )
--- a/prometheus/summary_test.go
+++ b/prometheus/summary_test.go
@@ -22,7 +22,7 @@
 	"testing/quick"
 	"time"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func benchmarkSummaryObserve(w int, b *testing.B) {
--- a/prometheus/value.go
+++ b/prometheus/value.go
@@ -20,7 +20,7 @@
 	"sort"
 	"sync/atomic"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/golang/protobuf/proto"
 )
--- a/text/bench_test.go
+++ b/text/bench_test.go
@@ -22,7 +22,7 @@
 
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 // Benchmarks to show how much penalty text format parsing actually inflicts.
--- a/text/create.go
+++ b/text/create.go
@@ -28,7 +28,7 @@
 	"strings"
 
 	"github.com/prometheus/client_golang/model"
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 // MetricFamilyToText converts a MetricFamily proto message into text format and
--- a/text/create_test.go
+++ b/text/create_test.go
@@ -20,7 +20,7 @@
 	"testing"
 
 	"github.com/golang/protobuf/proto"
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 func testCreate(t testing.TB) {
--- a/text/parse.go
+++ b/text/parse.go
@@ -22,7 +22,7 @@
 	"strconv"
 	"strings"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 
 	"github.com/golang/protobuf/proto"
 	"github.com/prometheus/client_golang/model"
--- a/text/parse_test.go
+++ b/text/parse_test.go
@@ -19,7 +19,7 @@
 	"testing"
 
 	"github.com/golang/protobuf/proto"
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 var parser Parser
--- a/text/proto.go
+++ b/text/proto.go
@@ -20,7 +20,7 @@
 	"github.com/golang/protobuf/proto"
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 
-	dto "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_golang/client_model"
 )
 
 // WriteProtoDelimited writes the MetricFamily to the writer in delimited
