--- a/prometheus/registry.go
+++ b/prometheus/registry.go
@@ -33,7 +33,7 @@
 	"strings"
 	"sync"
 
-	"bitbucket.org/ww/goautoneg"
+	"Godeps/_workspace/src/bitbucket.org/ww/goautoneg"
 	"code.google.com/p/goprotobuf/proto"
 
 	dto "github.com/prometheus/client_model/go"
--- a/prometheus/summary.go
+++ b/prometheus/summary.go
@@ -20,7 +20,7 @@
 	"sync"
 	"time"
 
-	"github.com/beorn7/perks/quantile"
+	"Godeps/_workspace/src/github.com/beorn7/perks/quantile"
 	"code.google.com/p/goprotobuf/proto"
 
 	dto "github.com/prometheus/client_model/go"
